xmcl-runtime/lib/services/InstallService.ts

```diff
@@ -165,7 +165,7 @@ export class InstallService extends AbstractService implements IInstallService {
       fabricMetaUrl = this.baseService.getApiSets()[0].url + '/fabric-meta'
     }
 
-    const response = await this.networkManager.request.get(`${fabricMetaUrl}/v2/versions/yarn`, {
```

Comment:
Sorry about this stupid error. I forgot to test the official api.

Can you change the line 163 

```ts
let fabricMetaUrl = DEFAULT_FABRIC_API
```

to

```ts
    let fabricMetaUrl = 'https://meta.fabricmc.net'
```
Otherwise it will break the BMCL api cases...

Reply:
ah, thanks for the heads up, will change